TestProjects/UniversalGraphicsTest_Foundation/Assets/Test/TestFilters/TestCaseFilters.asset

```diff
@@ -174,3 +174,21 @@ MonoBehaviour:
     XrSdk: 
     StereoModes: 0
     Reason: Needs fixing on DX12
+  - FilteredScene: {fileID: 0}
+    FilteredScenes:
+    - {fileID: 102900000, guid: 26800f4b3c38849428d29e93038a6084, type: 3}
+    ColorSpace: -1
+    BuildPlatform: -2
+    GraphicsDevice: 21
+    XrSdk: 
+    StereoModes: 0
+    Reason: Needs fix for MSAA+depth
+  - FilteredScene: {fileID: 0}
+    FilteredScenes:
+    - {fileID: 102900000, guid: 26800f4b3c38849428d29e93038a6084, type: 3}
+    ColorSpace: -1
+    BuildPlatform: -2
+    GraphicsDevice: 11
+    XrSdk: 
+    StereoModes: 0
+    Reason: Needs fix for MSAA+depth
```

Comment:
What are the exceptions being added here for which platforms?

Reply:
Excluding new camera motion vector test since there is a bug on vulkan with msaa and depth